.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 88
+max-line-length = 90
```

Comment:
Generally people should stick to the rules of formatting for a given project. I am not against this change (since `88` is arbitrary anyway) but I would like to know why did you need to introduce this modification?

Reply:
In our project, we have 2 formatting checks i.e `black` and `flake8` (even though I think either of them is enough), after formatting the code using `black` it allows lines upto 90 characters which is not the case with `flake8`. They both conflict, hence this change.